spec/features/events_spec.rb

```diff
@@ -22,6 +22,49 @@
     expect(page).to have_css("div#occupied_places")
   end
 
+  scenario "logged in as Organizer I want to be unable to send emails if there is any unclassified application left" do
+    login(:organizer)
+    @event.update!(max_participants: 1)
+    @pupil1 = FactoryGirl.create(:profile)
+    @pupil1.user.role = :pupil
+    @nilApplication = FactoryGirl.create(:application_letter, :event => @event, :user => @pupil1.user, :status => 2)
```

Comment:
As far as i know, the two are exactly the same, the second one is the "more recent" syntax. But I don't think we formally agreed to standards.